docs/archetypes/paas/overview.adoc

```diff
@@ -0,0 +1,238 @@
+= Archetype: App Service Environment + SQL Database
+:toc:
+:toc-placement: auto
+:toclevels: 2
+
+This archetype uses managed platform services. It provisions an Azure SQL Database and securely connects it to the workload's virtual network using a https://docs.microsoft.com/azure/virtual-network/virtual-network-service-endpoints-overview[Virtual Network service endpoint] for the data tier. The web and business tiers are provisioned using an https://docs.microsoft.com/azure/app-service/environment/intro[Azure App Service Environment] and securely connected to the workload virtual network.
+
+== Create your archetype configuration file
+
+The toolkit provides an example configuration file for the shared services environment. You will need to create a new version of this file before running your deployment.
+
+1. Navigate to the toolkit's link:../../../archetypes/paas[archetypes/paas] folder.
+1. Make a copy of the `archetype.test.json` and name the copy `archetype.json`.
+1. Edit `archetype.json` providing the subscription, organization, networking, and other configuration information that you want to use for your deployment. You will need to use some of the same values when creating the configuration for the shared services environment.
+
+If your copy of the toolkit is associated with the git repository, the link:../../.gitignore[.gitignore] file is set to prevent your deployment `archetype.json` file from being pushed to your code repository.
+
+== Deploy workload infrastructure
+
+All workload environments require a common set of operations, key vault, and virtual network resources before they can connect to the hub network and host workloads. The following steps will deploy these required resources.
+
+image:_media/workload-paas-empty.png[Before deploying the workload]
+
+=== Step 1: Deploy operations and monitoring resources
+
+*Required role: SysOps*
+
+This step provisions the operations and monitoring resources for the workload environment.
+
+Start the `la` deployment by running the following command in the terminal or command-line interface:
+
+.Docker
+[source,bash]
+python vdc.py create workload -path archetypes/paas/archetype.json -m la
+
+.Linux/OSX
+[source,bash]
+python3 vdc.py create workload -path archetypes/paas/archetype.json -m la
+
+.Windows
+[source,cmd]
+py vdc.py create workload -path archetypes/paas/archetype.json -m la
+
+This deployment creates the `{org}-{deployment-name}-la-rg` resource group that hosts the resources described in the following table.
+
+[options="header",cols="a,,"]
+|===
+| Resource | Type | Description
+
+| `{org}-{deployment-name}-la`
+| Log Analytics
+| Log Analytics instance for monitoring the workload's network.
+|===
+
+=== Step 2: Deploy Key Vault
+
+*Required role: SecOps*
+
+The next step is deploying the `kv` module, which creates a Key Vault for the shared services and generates the encryption keys that other resources will use in subsequent deployment steps.
```

Comment:
Remove shared services, since this is a PaaS doc.

Update the text:

The next step is deploying the `kv` module, which creates a Key Vault for the shared services and generates the encryption keys that other resources will use in subsequent deployment steps.

To:

The next step is deploying the `kv` module, which creates a Key Vault resource and store secrets defined in the Archetype as well as creating encryption keys (used in Azure Disk Encryption extension), that other resources will use in subsequent deployment steps.